docker-compose.yaml

```diff
@@ -81,10 +80,39 @@ services:
       - ./nginx:/etc/nginx/conf.d
       - static_files:/var/www/html/static
 
+  minio:
+    image: quay.io/minio/minio
+    ports:
+      - '9000:9000'
+      - '9001:9001'
+    #networks:
+    #  - api
+    volumes:
+      - 'minio_data:/data'
+    command: |
+      server /data --console-address ":9001"
+    environment:
+      - MINIO_ROOT_USER=sampleaccesskey
+      - MINIO_ROOT_PASSWORD=samplesecretkey
+      - MINIO_DEFAULT_BUCKETS=meshdb-join-form-log
+
+  # I hate computers: https://github.com/minio/minio/issues/4769
+  createbuckets:
+    image: minio/mc
+    depends_on:
+      - minio
+    entrypoint: >
+      /bin/sh -c "
+      /usr/bin/mc config host add myminio http://minio:9000 sampleaccesskey samplesecretkey;
+      /usr/bin/mc mb myminio/meshdb-join-form-log;
+      exit 0;
+      "
```

Comment:
Would it be better to use the AWS CLI here instead? Like you did in the join form PR

Reply:
Not sure if I could get the entrypoint to behave properly. This is just for development, so it's fine.